xbmc/addons/AddonDll.cpp

```diff
@@ -46,6 +48,7 @@ CAddonDll::CAddonDll(AddonProps props)
   m_pHelpers    = NULL;
   m_needsavedsettings = false;
   m_parentLib.clear();
+  memset(&m_interface, 0, sizeof(m_interface));
```

Comment:
Tried it, a memset with a `= 0` not possible. Have no alternative to them, the structure must complete in "C" style and also not want to add a "C++" constructor there.